### ISSUE TITLE
Add option to show on lock screen

### ISSUE THREAD

**Issue by PulsarFX** [home-assistant/android#3588]
**Is your feature request related to a problem? Please describe.**
Building a dashboard with home assistant app requires the screen to always stay on - or have some clever way to turn the (tablet) screen on and off.

**Describe the solution you'd like**
I'd like to have an option to show home assistant app on the lock screen. This enables the app to turn the display physically on and off.
This on the other hand can be used to trigger on/off based on a (motion) sensor state.

**Describe alternatives you've considered, if any**
BrowserMod allows to turn the screen black, but this keeps the backlight running (when not on an oled display, which my tablet does not have.)

**Additional context**
-

**Comment by dshokouhi** [comment]
do you know of any other app that lets you show the app on the lock screen? I am not sure how feasible this is but if there is an official API to do it then we can do it.

do you know that hte app offers a setting to keep the screen always on when the dashboard is open?

https://companion.home-assistant.io/docs/integrations/android-webview#keep-screen-on

**Comment by PulsarFX** [comment]
@dshokouhi  yes, Blitzer Plus https://play.google.com/store/apps/details?id=de.blitzer.plus  can blend itself over the whole lockscreen and hide all status notifications behind. It also can turn the screen on and off (or let it time out, dunno). 
In Settings > General, there are two options:
- "App in lock screen": "The app will show in the phone's lock screen upon deactivation of the display. The display needs to be manually activated once."
- "Automatically activate & deactivate display": "The display activates upon alert and deactivates afterwards".

When the app is running with the lock screen overlay, you can press home button (twice), then it falls back to the normal lock screen with the default notifications. 

There is a free version, too, which has about the same feature with turning screen on and off. The plus version was updated recently for newer android versions, and looks rather different.


Yes, I know about the "keep screen on", but this will cause burn in on the display after a while.
